src/backend/functions/work-packages-create.ts

```diff
@@ -31,32 +31,67 @@ export const createWorkPackage: Handler<FromSchema<typeof inputSchema>> = async
   } = body;
   // get the corresponding project so we can find the next wbs number
   // and what number work package this should be
-  const project = await prisma.project.findUnique({
-    where: { projectId },
+  const { carNumber, projectNumber, workPackageNumber } = projectWbsNum;
+
+  if (workPackageNumber !== 0) throw new TypeError('Given WBS Number is not for a project.');
+
+  const wbsElem = await prisma.wBS_Element.findUnique({
+    where: {
+      wbsNumber: {
+        carNumber,
+        projectNumber,
+        workPackageNumber
+      }
+    },
     include: {
-      wbsElement: true,
-      workPackages: { include: { wbsElement: true, dependencies: true } }
+      project: {
+        include: {
+          wbsElement: true,
```

Comment:
WbsElement is the parent of project, so by doing `include` within `project`, you’re saying get A > get B > get A and duplicating pulling the wbsElement

Reply:
Ah, I see. I'll remove that from the include